src/Monocle/Api/Jwt.hs

```diff
@@ -83,16 +83,21 @@ data LoginInUser = LoginInUser
   }
   deriving (Show, Eq, Ord)
 
+-- TODO - Add the Set-Cookie Header "Set-Cookie: JWT-Cookie=...."
```

Comment:
Actually, that should be the only way to set the cookie, such http-only cookie can't be set with javascript i think. For example, using: https://hackage.haskell.org/package/servant-auth-server-0.4.7.0/docs/Servant-Auth-Server.html#v:acceptLogin

Reply:
Yes, agree, with that solution the flow is working however I'm not happy with it and I'll prefer to let servant to set the proper header for the cookie. I'll try with your proposal.